src/librustc_passes/consts.rs

```diff
@@ -140,8 +140,7 @@ impl<'a, 'gcx> CheckCrateVisitor<'a, 'gcx> {
                     self.tcx.lint_node(CONST_ERR,
                                        expr.id,
                                        expr.span,
-                                       &format!("constant evaluation error: {}. This will \
-                                                 become a HARD ERROR in the future",
+                                       &format!("constant evaluation error: {}",
```

Comment:
@oli-obk 
I initially wanted to move `const_err` into the `future_incompatible` group based on this comment, but it looks like this comment was actually outdated and this lint is not going to be an error?

Reply:
Yea. These are essentially bug